app/controllers/placements/support/schools_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Placements::Support::SchoolsController < Placements::Support::ApplicationController
+  def new
+    @school = School.new
+  end
+
+  def check
+    return if school.valid? && !school.placements
+
+    school.errors.add(:urn, :taken) if school.placements
+    render :new
+  end
+
+  def create
+    school.placements = true
+    if school.save
+      redirect_to placements_support_organisations_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def school
+    @school ||=
+      begin
+        gias_school = GiasSchool.find_by(urn: gias_school_params)
+        if gias_school.blank?
+          School.new
```

Comment:
I might not have all the context here. I'm not sure about the necessity of this code. This should be a very rare edge case, but if it were to happen. Do we want to just create a school that is not linked to a Gias school?

Reply:
This is just to return a new `School` or `Placements::School` record back to the form when a `GiasSchool` can not be found. For example when a user enters nothing into the form input. As I then need to do some error handling on the new record.